globals-docs.json

```diff
@@ -283,13 +290,13 @@
     "SVGAnimationElement": "https://developer.mozilla.org/docs/Web/API/SVGAnimationElement",
     "SVGCircleElement": "https://developer.mozilla.org/docs/Web/API/SVGCircleElement",
     "SVGClipPathElement": "https://developer.mozilla.org/docs/Web/API/SVGClipPathElement",
-    "SVGColor": "https://developer.mozilla.org/",
+    "SVGColor": "https://hacks.mozilla.org/2017/10/containers-for-add-on-developers/",
```

Comment:
This one isn't quite right. Maybe it's time that we should just manually manage this list instead of relying on Google entirely for scrying the right URLs?

Reply:
It wouldn't be the end of the world... This is a bit more complete then what we have now anyway. Your call.

Reply:
👍 afaict no browser actually has a SVGColor global, so it should be fine to remove this line entirely

Reply:
patch incoming

Reply:
✅